app/javascript/components/patient/assessment/actions/ClearAssessments.js

```diff
@@ -45,28 +60,81 @@ class ClearAssessments extends React.Component {
+                  suspected after review of all of the monitoree&apos;s symptomatic reports. The &quot;Needs Review&quot; status will be changed to
+                  &quot;No&quot; for all reports. The record will move from the symptomatic line list to the asymptomatic or non-reporting line list as
+                  appropriate
+                  <b> unless a symptom onset date has been entered by a user.</b>
+                </p>
+              )}
+              {this.props.patient.isolation && (
+                <p>
+                  This will change any reports where the &quot;Needs Review&quot; column is &quot;Yes&quot; to &quot;No&quot;. If this case is currently under
+                  the &quot;Records Requiring Review&quot; line list, they will be moved to the &quot;Reporting&quot; or &quot;Non-Reporting&quot; line list as
+                  appropriate until a recovery definition is met.
+                </p>
+              )}
+            </React.Fragment>
           )}
           <Form.Group>
             <Form.Label>Please describe your reasoning:</Form.Label>
             <Form.Control as="textarea" rows="2" id="reasoning" onChange={this.handleChange} aria-label="Reasoning Text Area" />
           </Form.Group>
+          {this.props.patient.isolation &&
+            !this.props.patient.user_defined_symptom_onset &&
+            this.props.numPosLabs === 0 &&
+            (!this.props.assessment_id || this.props.onlySympAssessment) && (
+              <React.Fragment>
+                <p className="my-4">
+                  {`Marking ${
+                    this.props.assessment_id ? 'this report' : 'all reports'
+                  } as reviewed will result in the system populated Symptom Onset Date being cleared. Please provide a Symptom Onset Date or enter a positive lab result in order for this record to be eligible to appear on the Records Requiring Review line list.`}
+                </p>
+                <Form.Label className="nav-input-label">SYMPTOM ONSET</Form.Label>
+                <DateInput
+                  id="symptom_onset_mark_as_reviewed"
+                  date={this.state.symptom_onset}
+                  minDate={'2020-01-01'}
+                  maxDate={moment().add(30, 'days').format('YYYY-MM-DD')}
+                  onChange={date =>
+                    this.setState({
+                      symptom_onset: date,
+                      user_defined_symptom_onset: !!date,
+                    })
+                  }
+                  placement="bottom"
+                  customClass="form-control-lg mb-2"
+                  ariaLabel="Symptom Onset Date Input"
+                />
+                <div className="my-2"></div>
+                <FirstPositiveLab lab={this.state.first_positive_lab} onChange={lab => this.setState({ first_positive_lab: lab })} />
```

Comment:
Can you make this a `Modal` as well? Since it's smaller than the modal underneath it has a weird stacked look without the shading.

![image](https://user-images.githubusercontent.com/8235974/121263597-d5501b80-c883-11eb-9e0a-4ce65924c8f9.png)

It actually seems to functioning as a modal (I can't click the Submit button underneath), and so it's just an aesthetic issue.

Reply:
fixed: https://github.com/SaraAlert/SaraAlert/pull/861/commits/1670f66306ba72dce6246c84ead69a075c4a153f#diff-08fe027fbe0e1a0481ee292a8b7590c2b3aeeab2c63aeeab1dabb039918e9357R257